app.js

```diff
@@ -2,7 +2,8 @@ var Twit = require('twit');
 var path = require('path');
 var slackbot = require('node-slackbot');
 require('dotenv').config({
-	path: path.resolve('./env_variables')
+	path: path.resolve('./env_variables'),
+	silent: true
```

Comment:
I'm going to make a new issue to look into this and then close this PR in case anyone else wants to try to deploy to EB today (so we can see if it works)..